exes/mqtt-bridge/inc/tfc_to_external.hpp

```diff
@@ -1,80 +1,95 @@
 #pragma once
 
-#include <any>
-#include <cmath>
+#include <chrono>
+#include <concepts>
 #include <memory>
-#include <optional>
 #include <type_traits>
+#include <variant>
 #include <vector>
 
 #include <boost/asio.hpp>
 #include <sdbusplus/bus/match.hpp>
 
-#include <client.hpp>
-#include <config/broker.hpp>
-#include <config/spark_plug_b.hpp>
-#include <config/spark_plug_b_mock.hpp>
-#include <spark_plug_interface.hpp>
-#include <structs.hpp>
-#include <tfc/confman.hpp>
 #include <tfc/ipc.hpp>
-#include <tfc/ipc/details/dbus_client_iface_mock.hpp>
 #include <tfc/logger.hpp>
 
+#include <spark_plug_interface.hpp>
```

Comment:
do you have an alternative suggestion

Reply:
it's possible to depend on a generic interface in which spark plug b or possibly other communication protocols could depend upon, tfc to external would then depend upon that interface but in my opinion this is overengineering since we are only using spark plug b, it would be a good idea as soon as another external communication protocol is introduced